src/drivers/github.ts

```diff
@@ -111,7 +111,7 @@ export default defineDriver((_opts: GithubOptions) => {
 })
 
 async function fetchFiles(opts: GithubOptions) {
-  const prefix = withTrailingSlash(opts.dir)
+  const prefix = withTrailingSlash(opts.dir).replace(/^\//, '')
```

Comment:
Can you please use withoutLeadingSlash utility?

Reply:
Unfortunately `withoutLeadingSlash`'s behavior is to add a slash to empty strings so it doesn't really fit the bill here:
```js
withoutLeadingSlash('/docs') === 'docs'
withoutLeadingSlash('') === '/'
```

Alternatively we could use a ternary?
```js
opts.dir ? withoutLeadingSlash(withTrailingSlash(opts.dir)) : ''
```

Reply:
Hey! So I think you're mistaken? We need to remove the leading slash while keeping the trailing one. Otherwise, a config of `dir: 'docs'` would expose files like `docs_private/Secret.md`.

Reply:
Sorry i was mistaken because of title.

Reply:
Oh yikes my bad. Thanks for merging this in!